lib/msal-browser/src/app/IPublicClientApplication.ts

```diff
@@ -57,6 +57,7 @@ export interface IPublicClientApplication {
     initializeWrapperLibrary(sku: WrapperSKU, version: string): void;
     setNavigationClient(navigationClient: INavigationClient): void;
     getConfiguration(): BrowserConfiguration;
+    hydrateCache(result: AuthenticationResult, request: SilentRequest) : Promise<void>;
```

Comment:
hydrateCredentialCache?

Reply:
I'd prefer to keep it short but if there's inclination towards a different verb such as "saveTokens", "loadCache", etc. I'm open to suggestions/feedback

Reply:
This hydrates account and tokens, so I am good with the name. I do not think we ever needed `app meta data` and that is the only extra base type in cache for us.